doc/geany.txt

```diff
@@ -2715,6 +2715,10 @@ Don't use run script
     This can be useful if you already have a program running in the VTE
     like a Python console (e.g. ipython). Use this with care.
 
+Draw intense colors in bold fonts
+    Use bold fonts when printing characters that have the high-intensity
+    attribute.
```

Comment:
I used to think printing is valid even in context of screens, but ok, that's better.